dev/manual_tests/card_collection.dart

```diff
@@ -324,7 +324,7 @@ class CardCollectionState extends State<CardCollection> {
         child: new SingleChildScrollView(
           child: new Container(
             height: cardModel.height,
-            decoration: new BoxDecoration(backgroundColor: theme.primaryColor),
+            color: theme.primaryColor,
```

Comment:
In other APIs, "background" is the assumed position and "foreground" is called out explicitly.  For example, in CustomPaint, the `painter` is for the background and the `foregroundPainter` is for the foreground.  That's because operating on the background is 10x more common than operating on the foreground.